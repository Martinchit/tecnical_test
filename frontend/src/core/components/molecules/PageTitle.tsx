import React from 'react';
import { FlexBox } from '../atoms/FlexBox';
import { Text } from '../atoms/Text';

interface PageTitleProps {
  title: string;
}

export const PageTitle: React.FC<PageTitleProps> = ({ title }) => (
  <FlexBox flexDirection="column">
    <Text fontWeight="bold" fontSize={2.5}>
      {title}
    </Text>
  </FlexBox>
);
