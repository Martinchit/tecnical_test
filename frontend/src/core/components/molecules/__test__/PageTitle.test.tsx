import React from 'react';
import { shallow } from 'enzyme';

import { PageTitle } from '../PageTitle';

describe('PageTitle', () => {
  const title = 'Test';
  it('renders correctly', () => {
    const wrapper = shallow(<PageTitle title={title} />);
    expect(wrapper).toMatchSnapshot();
  });
});
