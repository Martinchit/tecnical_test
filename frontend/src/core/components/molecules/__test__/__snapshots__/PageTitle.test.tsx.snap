// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle renders correctly 1`] = `
<styled.div
  flexDirection="column"
>
  <styled.p
    fontSize={2.5}
    fontWeight="bold"
  >
    Test
  </styled.p>
</styled.div>
`;
